_posts/2017/2017-16-01-testing-async-interactions.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title: "Testing asynchronous interactions in distributed systems"
```

Comment:
With a small adjustment we could improve the headline: How to test asynchronous interactions in distributed systems

> http://coschedule.com/headline-analyzer#